index.html

```diff
@@ -0,0 +1,145 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@500;700&family=Open+Sans&display=swap"
+        rel="stylesheet">
+    <title>Resume_project1</title>
```

Comment:
The `title` element's content can be normal text and include spaces. It should generally be human-readable (since it's what users see when they bookmark a page, for example), so consider something more like
```suggestion
    <title>Project 1: Resume</title>
```
or
```suggestion
    <title>Resume (Project 1)</title>
```